sbin/cubeit

```diff
@@ -235,6 +238,24 @@ if [ "${#ROOTFS_LABEL}" -gt 16 ]; then
 	fi
 fi
 
+# Check if interactive mode will be used
+if [ -z ${CONFIG_FILES} ] || ([ -n "$INTERACTIVE_MODE" ] && [ "$INTERACTIVE_MODE" -eq 1 ]); then
```

Comment:
We want to support interactive installer for both host cross install and on-target installer image, so I think we should support this interactive mode in both.